README.md

```diff
@@ -1,2 +1,43 @@
+| **Platform Launch** | {ommitted} | If context has already been established, naming the capability in the sentence may be obsolete and can be removed entirely. | If the extensions, tags, and systems you are going to deploy through **Platform Launch** are very similar across your sites or applications, you might want to include them in the same property. | If the extensions, tags, and systems you are going to deploy are very similar across your sites or applications, you might want to include them in the same property. |
+| **An Adobe Experience Platform Launch {noun}** | a tags-enabled {noun} | When tags is providing the functionality to the noun, the hyphenated 'enabled' adds clarity to the sentence.  | This extension provides the functionality for adding the 'Media' tracker instance to **an Adobe Experience Platform Launch site** or project.  | This extension provides the functionality for adding the `Media` tracker instance to **a tags-enabled site** or project.   |
+| **The Launch runtime**  |  The tag runtime | Use singular version of Tag and "The" as an article when talking about a general concept/process rather than individual runtime events.  | **The Adobe Experience Platform Launch runtime** (delivered to clients' websites) is designed to support the following browsers:  | ****The tag runtime**** in Adobe Experience Platform (delivered to clients' websites) is compatible with multiple browsers:  |
+| **A Launch property**  | A tag property  | Use the singular version of 'tag' when writing "tag {noun}" in a sentence. | When users install an extension to an Adobe Experience Platform **Launch property**, they will be shown the extension configuration view which your extension will provide.  | When users install an extension to **a tag property**, they will be shown the extension configuration view which your extension will provide.  |
+| **the {noun} extension in Platform Launch**  |  the {noun} tag extension in Platform | The name of the extension will precede the tag qualifier. Later references to tags (Launch) are removed once context has been established.  | Learn about **the Adobe ContextHub extension in Adobe Experience Platform Launch**. | Learn about **the Adobe ContextHub tag extension in Adobe Experience Platform**.  |
+| **Adobe Experience Platform Launch {verb} ...**  | Adobe Experience Platform {verb} | Launch is not always necessary and can be omitted. When the sentence refers to Launch as a service it can often be replaced by referring to Platform. | **Adobe Experience Platform Launch provides** a unified catalog where users can view tag extensions that are available for installation. | **Adobe Experience Platform provides** a unified catalog where users can view tag extensions that are available for installation. |
```

Comment:
Is this necessary? This isn't a term change at all...just reiterating how to handle Launch as a service versus individual capabilities. Remove or incorporate into a separate section.